Bender.yml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: apb_adv_timer
+  authors: 
+    - "Florent Rotenberg <florent.rotenberg@greenwaves-technologies.com>"
+    - "Yao Zhang <yao.zhang@greenwaves-technologies.com>"
+    - "Antonio Pullini <pullinia@iis.ee.ethz.ch>"
+    - "Davide Schiavone <pschiavo@iis.ee.ethz.ch>"
+
+sources:
+  - files:
+      - rtl/adv_timer_apb_if.sv
+      - rtl/comparator.sv
+      - rtl/lut_4x4.sv
+      - rtl/out_filter.sv
+      - rtl/up_down_counter.sv
+      - rtl/input_stage.sv
+      - rtl/prescaler.sv
+      - rtl/apb_adv_timer.sv
+      - rtl/timer_cntrl.sv
+      - rtl/timer_module.sv
```

Comment:
These files are not properly ordered: `apb_adv_timer` instantiates `timer_module`, which in turn instantiates `timer_cntrl` and many more. So `apb_adv_timer` should come last.  (This is not problematic in all tools, but some will fail to compile, so it must be fixed for maximum compabitility.)

I suggest to group files into *levels*, e.g., [as done here](https://github.com/pulp-platform/common_cells/blob/a06e4beb08b8c0a5ef2c85d9afcfe0d96be76f30/Bender.yml#L20-L22). This is a simple and biunique ordering that works with any tool.

Please also fix `src_files.yml`.